k8s.gcr.io/images/k8s-staging-e2e-test-images/images.yaml

```diff
@@ -0,0 +1,21 @@
+- name: agnhost
```

Comment:
Why only these many images? where are the rest of the images?
Did we run e2e with the staging images? can we just run the tests modifying the repo to staging repo to see if they really works :)

Reply:
Plenty of questions to unpack here. :)

1. Currently, the Image Promoter was set to work with the Conformance Images. I've left some comments here: https://github.com/kubernetes/kubernetes/blob/30058c9d356c5c10c6273b58a41057d9ab252815/test/images/image-util.sh#L164 . Basically, for the 1st implementation, we decided to only go with the Conformance Images, since those are most commonly used in tests and those are the images that are commonly updated, and some non-conformance images take an extremely long time to build.
2. The rest of the images we could probably add in the future, maybe, if there's a need for it, but it's a lower priority item (see above).
3. We have not yet. I could send a PR to check that, sounds like a good idea. :)

Reply:
👍

Reply:
It seems that it was a good idea to actually check those images. It seems that some issue slipped through, especially due to the lack of logging (a ``-x`` option would help with that). Basically, the job builds ``all-conformance`` images, but incidentally, it just publishes the ``agnhost image`` under different names. Increasing the verbosity, I could see why:

    ++ docker manifest create --amend claudiubelu/jessie-dnsutils:1.2 claudiubelu/agnhost-amd64:2.9 claudiubelu/agnhost-arm:2.9 claudiubelu/agnhost-arm64:2.9 claudiubelu/agnhost-ppc64le:2.9 claudiubelu/agnhost-s390x:2.9 claudiubelu/jessie-dnsutils-amd64:1.2 claudiubelu/jessie-dnsutils-arm:1.2 claudiubelu/jessie-dnsutils-arm64:1.2 claudiubelu/jessie-dnsutils-ppc64le:1.2 claudiubelu/jessie-dnsutils-s390x:1.2
    Created manifest list docker.io/claudiubelu/jessie-dnsutils:1.2

Seems that the ``manifest`` list that's being used in the ``image-util.sh`` is stateful, and the images just pile on in that list. That list needs to be reset. Will send a PR soon, but the fix is simple.

Reply:
Sent PR: https://github.com/kubernetes/kubernetes/pull/87647

Reply:
This PR tests the staging images: https://github.com/kubernetes/kubernetes/pull/87607

It seems there was an issue with the kitten and nautilus images. Because the agnhost version was bumped a few times before the Image Centralization part 4 merged, the kitten and nautilus base versions were never updated, and thus, didn't have the test-webserver functionality they needed. This PR fixes that: https://github.com/kubernetes/kubernetes/pull/87722

I think after we merge that and update the digests here, we can merge this PR.